birdhouse/optional-components/weaver/config/weaver/weaver.ini.template

```diff
@@ -0,0 +1,156 @@
+# Weaver settings configuration
+# See original reference for details:
+#   https://github.com/crim-ca/weaver/blob/master/config/weaver.ini.example
+#
+
+###
+# app configuration
+# http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html
+###
+
+[app:main]
+use = egg:weaver
+
+pyramid.reload_templates = false
+pyramid.debug_authorization = false
+pyramid.debug_notfound = false
+pyramid.debug_routematch = false
+pyramid.default_locale_name = en
+
+# mongodb
+mongodb.host = mongodb
+mongodb.port = 27017
+mongodb.db_name = weaver
+
+# --- Weaver Configuration ---
+weaver.configuration = ${WEAVER_CONFIG}
+weaver.url = https://${PAVICS_FQDN}/weaver
```

Comment:
/weaver is hardcoded but nginx is configured with `${WEAVER_CONFIG}` and `${WEAVER_MANAGER_NAME}`

Reply:
Good point. I also removed the `/${WEAVER_CONFIG}` location as that will be weird once some kind of hybrid EMS/ADES config gets introduced.